compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -834,6 +834,10 @@
   [x]
   (native/raw "__value = make_box(~{ x }->type == object_type::volatile_);"))
 
+; Delay and force
+(defn delay [exp] (native/raw "__value = make_box<obj::delay>(~{ exp });"))
+(defn force [exp] (native/raw "__value = try_object<obj::delay>(~{ exp })->deref();"))
```

Comment:
Good work getting something going here.

1. The comment doesn't provide anything useful; let's remove it
2. We'll need to rewrite delay to be a macro, in order for any of this to work properly; we'll talk about that on Monday
3. The `force` implementation looks good, with two stylistic requests
  a. We don't use `exp` for param names anywhere else; for general objects, we use `o`, but you could use `d` if you're expecting a delay
  b. We always put fn bodies on a new line, not on the same line as the args